mechanical/mechanical-main.md

```diff
@@ -6,6 +6,6 @@
 - Common parts suppliers
 
 ### CAD (Computer-Aided Design)
-- OnShape Setup
+- [Onshape - Getting Started](/mechanical/cad/cad-getting-started.md)

```

Comment:
```suggestion
- [Onshape - Getting Started](./cad/cad-getting-started.md)

```
Without the `./` may also work, but this is more explicit
This file is already in the `mechanical` directory so this link is breaking